spec/features/idv/steps/enter_code_step_spec.rb

```diff
@@ -154,7 +154,12 @@
 
       expect(current_path).to eq idv_verify_by_mail_enter_code_path
       expect(page).to have_content t('idv.gpo.alert_info')
-      expect(page).to have_content strip_tags(t('idv.gpo.change_to_verification_code_html'))
+      # The following test ensures that removed content is actually removed. The text
+      # is hardcoded since the i18n entries have been removed. Once we're satisfied,
+      # this can be removed.
+      expect(page).not_to have_content(
+        'The one-time code from your letter is now referred to as verification code',
+      )
```

Comment:
It's probably fine to just have no assertion at all, rather than the assertion of absence of content?

```suggestion
```

Otherwise, can you make sure we have a ticket and an estimated ETA for removal?

Reply:
I'm having trouble running specs locally (🤬) so I'm letting CI do it for me. I'm happy to delete prior to merging.

Reply:
Removed in 4b022a768